### PR TITLE
IParagraph.AddText implementation

### PR DESCRIPTION
**Pull request by sergsuhanov**
Adding text creates IPortion entries so you can change the font and color.

```
var newPar = paragraphs.Add();
newPar.Text = "Text";
newPar.AddText("Text2");

var last = newPar.Portions.Last();
var font = last.Font;
font.IsItalic = true;
font.Size = 18;
```

### CODE REVIEW

File: ShapeCrawler/AutoShapes/IParagraph.cs [hunk 1]
```diff
@@ -9,6 +9,11 @@ namespace ShapeCrawler
     /// </summary>
     public interface IParagraph
     {
+        /// <summary>
+        /// Add text to paragraph.
+        /// </summary>
+        void AddText(string text);
```
Comment (adamshakhabov) [inline ShapeCrawler/AutoShapes/IParagraph.cs:15]: @sergsuhanov, thank you for the PR.

What about moving this to `IPortionCollection` level? I think something like `newPr.Portions.Add("Text2")` would be more consistent.

Comment (sergsuhanov) [reply ShapeCrawler/AutoShapes/IParagraph.cs:15]: hi, maybe
IPortion maps to A.Run and when we add a text separator we use A.Break.

`var lastElement = this.AParagraph.Where(p => p is A.Run || p is A.Break).Last(); // for inset after this`
I think that at the paragraph level, working with text is more logical.

Comment (adamshakhabov) [reply ShapeCrawler/AutoShapes/IParagraph.cs:15]: Got it. Then what about rename `AddText()` -> `AddPortion()`

Comment (sergsuhanov) [reply ShapeCrawler/AutoShapes/IParagraph.cs]: Ok, sure

**Review by adamshakhabov — APPROVED** [review]

**Pull request merged**